ajax/debug.php

```diff
@@ -43,14 +43,18 @@
     die();
 }
 
-// No need to save session data. Cannot use Session::writeClose because it doesn't do anything in debug mode
-session_write_close();
 \Glpi\Debug\Profiler::getInstance()->disable();
 
 if (isset($_GET['ajax_id'])) {
     // Get debug data for a specific ajax call
     $ajax_id = $_GET['ajax_id'];
-    $profile = \Glpi\Debug\Profile::load($ajax_id);
+    $profile = \Glpi\Debug\Profile::pull($ajax_id);
+
+    // Close session ASAP to not block other requests.
+    // DO NOT do it before call to `\Glpi\Debug\Profile::pull()`,
+    // as we have to delete profile from `$_SESSION` during the pull operation.
+    session_write_close();
```

Comment:
This was previously called each time, but now this is conditionned. Won't this be a problem?

Reply:
Outside this if block, there is nothing to do, so script will exit almost instantly, releasing the session file lock.